UserGuide.md

```diff
@@ -233,6 +237,73 @@ Unfortunately, there is no way to get around this in Java.
 Gson can serialize collection of arbitrary objects but can not deserialize from it, because there is no way for the user to indicate the type of the resulting object. Instead, while deserializing, the Collection must be of a specific, generic type.
 This makes sense, and is rarely a problem when following good Java coding practices.
 
+### <a name="TOC-Maps-Examples"></a>Maps Examples
+
+Gson by default serializes `java.util.Map` implementations as JSON object. Because JSON objects only support strings as member names, Gson converts the Map keys to strings by calling `toString()` on them, and using `"null"` for `null` keys:
```

Comment:
Perhaps clearer: "Gson by default serializes any `java.util.Map` implementation as a JSON object." To make it clear that one map becomes one JSON object.